__tests__/config/default.yaml

```diff
@@ -1,4 +1,4 @@
-keyVault:
+propertyVolume:
```

Comment:
this still looks weird to me,
The calling code would look like:
```
config.get('propertyVolume.idam.client-secret')
```

I think this would look much better:
```
config.get('secrets.idam.client-password')
```

Reply:
sure ... secrets work.